docs/sphinx/source/site_data.py

```diff
@@ -91,10 +91,7 @@ def test_set_name_to_parameters_and_image():
     test_set_names = list(utils.get_filenames_in_directory(utils.TEST_SETS_DIR))
     test_set_names.sort()
     for name in test_set_names:
-        info = {}
-        for idx, parameters in utils.read_iv_curve_parameter_sets(f'{utils.TEST_SETS_DIR}/{name}').items():
-            info[idx] = {'title': f'Case {idx}',
-                         'parameters': list(map(mp.nstr, parameters[:-1])), # exclude cells_in_series
-                         'image_path': f'./_images/test_cases/{utils.make_iv_curve_name(name, idx)}.png'}
-        mapping[name] = info
+        data = utils.read_iv_curve_parameter_sets(f'{utils.TEST_SETS_DIR}/{name}')
+        # limit params to params[:-1] to not publish cells_in_series
+        mapping[name] = [[idx, *params[:-1]] for idx, params in data.items()]
```

Comment:
```suggestion
        mapping[name] = [[idx, *params[:-1]] for idx, params in data.items()]
        # stringify floats with mp.nstr to write saturation_current in scientific notation
        mapping[name] = [list(map(mp.nstr, row)) for row in mapping[name]]
```
Currently the table looks like this:
![image](https://user-images.githubusercontent.com/41020789/225365311-207d7c7e-aabd-49e7-9c04-f4a280c1bbc5.png)
By adding this line, it will look like this:
![image](https://user-images.githubusercontent.com/41020789/225365854-411d485e-0ef3-4085-98d9-b445e1868814.png)